docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -17,86 +16,130 @@
    openvino_docs_MO_DG_prepare_model_Model_Optimizer_FAQ
 
 
-Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, 
-performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
+To convert a model to IR, you can use the following command:
 
+.. tab-set::
 
-To use it, you need a pre-trained deep learning model in one of the supported formats: 
-TensorFlow, PyTorch, ONNX, TensorFlow Lite, and PaddlePaddle (OpenVINO support for Apache MXNet, Caffe, and Kaldi is currently 
-being deprecated and will be removed entirely in the future). Model Optimizer converts the model to the OpenVINO Intermediate Representation format (IR), 
-which you can infer later with :doc:`OpenVINO™ Runtime <openvino_docs_OV_UG_OV_Runtime_User_Guide>`.
+    .. tab-item:: mo CLI tool
+       :sync: cli-tool
 
-Note that Model Optimizer does not infer models.
+       .. code-block:: sh
 
-The figure below illustrates the typical workflow for deploying a trained deep learning model:
+          mo --input_model INPUT_MODEL
 
-.. image:: _static/images/BASIC_FLOW_MO_simplified.svg
+    .. tab-item:: mo Python API
+       :sync: mo-python-api
 
-where IR is a pair of files describing the model:
+       .. code-block:: python
 
-* ``.xml`` - Describes the network topology.
+          from openvino.tools.mo import convert_model
+          ov_model = convert_model(INPUT_MODEL)
 
-* ``.bin`` - Contains the weights and biases binary data.
+       .. note::
 
-The OpenVINO IR can be additionally optimized for inference by :doc:`Post-training optimization <pot_introduction>` that applies post-training quantization methods.
+          You do not need to specify the ``input_model`` parameter explicitly. The parameter can be set as an unnamed argument and then it will be used as input ``model``. Therefore, both ``convert_model(input_model=model)`` and ``convert_model(model)`` will have the same effect.
```

Comment:
It hasn't been completely fixed:

![image](https://github.com/openvinotoolkit/openvino/assets/29092471/c1e8932e-e3e0-443a-8604-56202c9456ef)

You see that there is no `input_model` parameter in the example. It is better to omit this note completely.

Reply:
I implemented the changes with [fc9ce04](https://github.com/openvinotoolkit/openvino/pull/17454/commits/fc9ce049bc7d3622434764522eabd9afc4fe7184) commit.